src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1054,17 +1076,17 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
         goto err_lru_cleanup;
     }
 
-    status = uct_ib_mlx5_devx_query_lag(md, &lag_state, &port_select_mode);
+    status = uct_ib_mlx5_devx_query_lag(md, &lag_state);
     if (status != UCS_OK) {
         dev->lag_level   = 0;
-        port_select_mode = UCT_IB_LAG_INVALID_MODE;
     } else if (lag_state == 0) {
         dev->lag_level = 1;
     } else {
         dev->lag_level = UCT_IB_MLX5DV_GET(cmd_hca_cap, cap, num_lag_ports);
+        uct_ib_mlx5_devx_query_port_select(md, &port_select_mode);
```

Comment:
can we remove port_select_mode local variable, and assign directly to md->port_select_mode ?